components/CostsTable.vue

```diff
@@ -173,36 +162,56 @@ const scenariosToDisplay = computed(() => {
 
 const scenarioLabel = (scenario: Scenario) => appStore.getScenarioAxisLabel(scenario);
 
-const vslVariesByScenario = computed(() => {
-  return props.scenarios.some(scenario => appStore.getScenarioLifeValue(scenario) !== appStore.getScenarioLifeValue(props.scenarios[0]));
-});
-
-const vslLabel = (scenario: Scenario) => {
-  const vsl = appStore.getScenarioLifeValue(scenario);
-  return `${commaSeparatedNumber(vsl)} Int'l$`;
+const withSign = (value: string): string => {
+  const sign = Number.parseInt(value) < 0 ? "-" : "+";
+  return `${props.diffing ? sign : ""}${value.replace("-", "")}`;
 };
 
-const displayValue = (scenario: Scenario, costId: string): string | undefined => {
+const displayValue = (scenario: Scenario, costId: string, metricId: string): string | undefined => {
   const cost = appStore.getScenarioCostById(scenario, costId)!;
-  const valueInDollarTerms = props.diffing ? diffAgainstBaseline(cost) : getDollarValueFromCost(cost);
-  if (valueInDollarTerms === undefined) {
+  const val = props.diffing ? diffAgainstBaseline(cost, metricId) : getValueFromCost(cost, metricId);
+  if (val === undefined) {
     return;
   }
+  if (metricId !== USD_METRIC) {
+    const { amount, unit } = abbreviateMillions(val / 1000_000, true, 1);
+    return `${withSign(amount)}${unit}`;
+  }
   switch (appStore.preferences.costBasis) {
     case CostBasis.PercentGDP:
     {
-      const percentOfGdp = costAsPercentOfGdp(valueInDollarTerms, scenario.result.data?.gdp);
-      return `${humanReadablePercentOfGdp(percentOfGdp).percent}%`;
+      const percentOfGdp = costAsPercentOfGdp(val, scenario.result.data?.gdp);
+      return `${withSign(humanReadablePercentOfGdp(percentOfGdp).percent)}%`;
     }
     case CostBasis.USD:
     {
-      return Math.abs(valueInDollarTerms) > 10_000
-        ? (Math.round(valueInDollarTerms / 1000) * 1000).toLocaleString()
+      const amount = Math.abs(val) > 10_000
+        ? (Math.round(val / 1000) * 1000).toLocaleString()
         : new Intl.NumberFormat("en-US", {
             maximumSignificantDigits: 1,
-          }).format(valueInDollarTerms);
+          }).format(val);
+      return withSign(amount);
+    }
+  }
+};
+
+const displayDeaths = (scenario: Scenario): string | undefined => {
+  const series = scenario.result.data!.time_series[CUMULATIVE_DEATHS_SERIES_ID];
+  const totalDeaths = series.at(-1);
+  if (totalDeaths === undefined) {
+    return;
+  }
+  if (props.diffing) {
+    const baselineSeries = appStore.baselineScenario?.result.data!.time_series[CUMULATIVE_DEATHS_SERIES_ID];
+    const baselineTotalDeaths = baselineSeries?.at(-1) || 0;
+    if (baselineTotalDeaths === undefined) {
+      return;
     }
+    const { amount, unit } = abbreviateMillions((totalDeaths - baselineTotalDeaths) / 1000_000, true, 1);
+    return `${withSign(amount)}${unit}`;
   }
+  const { amount, unit } = abbreviateMillions(totalDeaths / 1000_000, true, 1);
```

Comment:
This feels a little bit odd. We're convering a value which isn't in millions, into millions, then calling `abbreviateMillions` on it...! Because we had the `abbreviateMillions` method around anyway to help with abbreviating the dollar amounts - which are in millions..?

Now that we have non-million amounts to abbreviate, it might feel more natural to convert the dollar amount to its corresponding real value and just have `abbreviate`..?

We could also use something like numeral.js to take care of the formatting.

Reply:
> We're convering a value which isn't in millions, into millions, then calling abbreviateMillions on it...!

I always like to leave a little joke in my PRs, to keep them light